cli/cmd/release_lint.go

```diff
@@ -13,18 +14,19 @@ func (r *runners) InitReleaseLint(parent *cobra.Command) {
 		Use:   "lint",
 		Short: "Lint a YAML",
 		Long:  "Lint a YAML",
+		SilenceUsage: true,
```

Comment:
This keeps it from printing the command usage on a failure

Reply:
Nice!